doc/PLUGINS.md

```diff
@@ -247,6 +247,92 @@ forms:
 `jcon fd <error_fd_to_jsonrpc>:`, `plugin-manager`;
 4. `log` is the context of the original log entry.
 
+#### `forward_event`
+
+A notification for topic `forward_event` is sent every time the status
+of a forward payment is set. The json format is same as the API
+`listforwards`.
+
+```json
+{
+  "forward_event": {
+  "payment_hash": "f5a6a059a25d1e329d9b094aeeec8c2191ca037d3f5b0662e21ae850debe8ea2",
+  "in_channel": "103x2x1",
+  "out_channel": "103x1x1",
+  "in_msatoshi": 100001001,
+  "in_msat": "100001001msat",
+  "out_msatoshi": 100000000,
+  "out_msat": "100000000msat",
+  "fee": 1001,
+  "fee_msat": "1001msat",
+  "status": "settled",
```

Comment:
The rest of your document refers to `FORWARD_OFFERED`, `FORWARD_SETTLED`, etc.  Which one is actually used on the actual plugin interface?  Please use what is actually visible to plugins, as this is `PLUGINS.md`, and avoid reference to internal defines that plugins need not know about.

Reply:
Thank you, I'll change `FORWARD_SETTLED` to `"settled"`, etc.